src/pile.js

```diff
@@ -1032,6 +1032,54 @@ const createPile = (
     setCover(newCover);
   };
 
+  let labelGraphics;
+  let labelTextContainer;
+
+  const drawLabel = (labels, colors, texts) => {
+    if (isPositioning || isScaling) {
+      postPilePositionAnimation.set('drawLabel', () => {
+        drawLabel(labels, colors, texts);
+      });
+      return;
+    }
+
+    if (!labelGraphics || !labelTextContainer) {
+      labelGraphics = new PIXI.Graphics();
+      labelTextContainer = new PIXI.Container();
```

Comment:
Do we really need another Container here? I think we can add the text directly to `labelGraphics`.